aten/src/ATen/native/Activation.cpp

```diff
@@ -114,26 +115,31 @@ void inline prelu_cpu_kernel_multi_weights(
   int64_t input_stride0,
   int64_t input_stride1) {
 
-  int64_t i, j, k;
   int64_t input_numel = input.numel();
   scalar_t* result_data = result.data<scalar_t>();
   scalar_t* input_data = input.data<scalar_t>();
   scalar_t* weight_data = weight.data<scalar_t>();
 
-  #pragma omp parallel for private(i,j,k) if (input.numel() > 1000)
-  for (i = 0; i < input_dim0_size; ++i) {
-    int64_t offset = i * channel_size * input_stride1;
-    scalar_t* n_input_data = input_data + offset;
-    scalar_t* n_result_data = result_data + offset;
-    for (j = 0; j < channel_size; ++j) {
-      for (k = 0; k < input_stride1; ++k) {
-        // to allow for compiler optimization, here splitting into two lines:
-        scalar_t w = (n_input_data[k] > 0) ? scalar_t(1) : weight_data[j];
-        n_result_data[k] = w * n_input_data[k];
+  auto loop = [&](int64_t start, int64_t end) {
+    for (auto i = 0; i < end; ++i) {
```

Comment:
tests are failing probably because of `0` -> `start` here. I'm worried we might have missed other places too. If you compile it with unused variable warnings - does it produce more warnings?

Reply:
oops, missed one, thanks - I am definitely going to audit all the places once more time